ext/XS-APItest/APItest.xs

```diff
@@ -4367,6 +4378,21 @@ CODE:
 OUTPUT:
     RETVAL
 
+pthread_t
+make_signal_thread()
+CODE:
+    if (pthread_create(&RETVAL, NULL, signal_thread_start, NULL) != 0)
+        XSRETURN_EMPTY;
```

Comment:
Why return empty list instead of `&PL_sv_undef` or `&PL_sv_no` or SvPV "" empty str or less perfect, SvIV -1 or SvIV 0? I know `pp_entersub` will cast empty list to 1 x `&PL_sv_undef`  in `G_SCALAR` cxt, but why mk `pp_entersub do the cast?